src/HanziWriter.js

```diff
@@ -40,6 +40,7 @@ const defaultOptions = {
   highlightColor: '#AAF',
   outlineColor: '#DDD',
   drawingColor: '#333',
+  highlightCompleteColor: '#0ff',
```

Comment:
Could this be `null` by default? That way it won't change behavior for existing users

Reply:
Yes, of course.